mmengine/structures/instance_data.py

```diff
@@ -67,100 +67,93 @@ class InstanceData(BaseDataElement):
         >>> print(instance_data)
         <InstanceData(
             META INFORMATION
-            pad_shape: (800, 1196, 3)
-            img_shape: (800, 1216, 3)
+            img_shape: (800, 1196, 3)
+            pad_shape: (800, 1216, 3)
             DATA FIELDS
+            det_scores: tensor([0.8000, 0.7000])
             det_labels: tensor([2, 3])
-            det_scores: tensor([0.8, 0.7000])
-            bboxes: tensor([[0.4997, 0.7707, 0.0595, 0.4188],
-                [0.8101, 0.3105, 0.5123, 0.6263]])
             polygons: [[1, 2, 3, 4], [5, 6, 7, 8]]
-            ) at 0x7fb492de6280>
+            bboxes: tensor([[0.4997, 0.7707, 0.0595, 0.4188],
```

Comment:
The order of DATA FIELDS is not consistent so the order of `bboxes` and `det_scores` can be left unchanged.